src/page-modules/assistant/utils.ts

```diff
@@ -136,3 +119,12 @@ export function setTransportModeFilters(
       .map((filter) => filter.id) ?? null
   );
 }
+
+export const getNoticesForLeg = (leg: LegFragment) =>
+  filterNotices([
+    ...(leg.line?.notices || []),
+    ...(leg.serviceJourney?.notices || []),
+    ...(leg.serviceJourney?.journeyPattern?.notices || []),
+    ...(leg.fromEstimatedCall?.notices || []),
+    ...(leg.toEstimatedCall?.notices || []),
```

Comment:
Ok, I see now this is included now here, there is another task I am working on to have this attribute included.